pytest_html/plugin.py

```diff
@@ -344,14 +381,21 @@ def generate_summary_item(self):
             unicode_doc = unicode_doc.decode('utf-8')
         return unicode_doc
 
-    def _save_report(self, report_content, self_contained_html):
+    def _save_report(self, report_content):
         dir_name = os.path.dirname(self.logfile)
-        if not os.path.exists(dir_name):
-            os.makedirs(dir_name)
+        assets_dir = os.path.join(dir_name, 'assets')
```

Comment:
Well, now I saw that in fact it is used in many places, but in different classes, so it would have to be an argument of TestResult and HTMLReport, so it would have no difference.